From 340362965f91e0aefa503d47c8a1304efd647b67 Mon Sep 17 00:00:00 2001
From: Jianqun Xu <jay.xu@rock-chips.com>
Date: Tue, 13 Dec 2016 17:12:20 +0800
Subject: [PATCH] ARM64: dts: rockchip: remove isp support for rk3399 evb3

Enable isp will cause system to halt on android booting, which
is caused by wrong voltage for vdd_gpu.

isp will set gpio directly, unfortunatly, it sets the enable gpio
of dc-dc for gpu.

There has no good solution for isp to fix this issue, so let's
disable the isp node for rk3399 evb3 firstly to make board works
well.

Change-Id: I5dd8c367a192ce793ed388aaf143125550e90621
Signed-off-by: Jianqun Xu <jay.xu@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-evb-rev3-android.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3-android.dts b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3-android.dts
index 3b3ff7cd52b6..c0e513819c59 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3-android.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3-android.dts
@@ -153,7 +153,7 @@
 		rockchip,camera-module-mipi-dphy-index = <0>;
 	};
 };
-
+/*
 &isp0 {
 	status = "okay";
 };
@@ -165,4 +165,4 @@
 &cif_isp0 {
 	status = "okay";
 	rockchip,camera-modules-attached = <&camera0>;
-};
+};*/
-- 
2.35.3

